src/core/polymer.cpp

```diff
@@ -186,6 +187,11 @@ draw_polymer_positions(PartCfg &partCfg, int const n_polymers,
         } else if (not positions[p].empty()) {
           /* Go back one position and try again */
           positions[p].pop_back();
+          rejections++;
```

Comment:
Can you please explain why this is needed? Couldn't you just increase `attempts_poly`?

Reply:
Ah sorry, when the first one has fixed position. But there has to be a more elegant solution than this...

Reply:
this while loop doesn't increment `attempts_poly`, therefore in high density systems, you can get stuck in an infinite cycle of deleting the previous bead and adding a new one at the same position